cmake/atmosphericphysicsConfig.cmake.in

```diff
@@ -0,0 +1,5 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@_Exports.cmake")
+
+check_required_components("@PROJECT_NAME@")
```

Comment:
I don't believe this file is necessary. This is only used to make an install target and atmospheric physics won't be installed with cmake as far as I'm aware

Reply:
This makes sense, removed.